site/src/modules/resources/AgentStatus.tsx

```diff
@@ -296,6 +305,44 @@ export const SubAgentStatus: FC<SubAgentStatusProps> = ({ agent }) => {
 	);
 };
 
+const DevcontainerStartError: FC<AgentStatusProps> = ({ agent }) => {
```

Comment:
I would suggest only passing the troubleshooting URL to make it clear why parent agent is relevant.

PS. I presume we don't copy it from the parent at present?